core/src/main/java/com/radixdlt/consensus/sync/GetVerticesRequest.java

```diff
@@ -67,5 +67,5 @@
 import com.google.common.hash.HashCode;
 import com.radixdlt.consensus.event.RemoteEvent;
 
-/** Parameters for a local get vertices request */
+/** Parameters for a local get vertices request. Also, could be scheduled locally in tests. */
```

Comment:
I don't think this comment is correct - we don't schedule it anywhere in tests, unless I'm missing something?
The first part of this comment seems wrong too: "Parameters for a local get vertices request".
I'd change it to: "A request to fetch up to _count_ vertices starting at _vertexId_ from a remote peer.".

Reply:
Fixed